.github/workflows/linter.yml

```diff
@@ -56,4 +56,8 @@ jobs:
           VALIDATE_DOCKERFILE: false
           VALIDATE_DOCKERFILE_HADOLINT: false
           VALIDATE_PYTHON_FLAKE8: false
+          VALIDATE_JSCPD: false
+          VALIDATE_KOTLIN: false
```

Comment:
Can you set kotlin as true?

Reply:
Okay. I was just afraid of getting errors.